docs/docs-beta/docs/tutorials/category-two/prompt-engineering/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Prompt engineering
+description: Learn how to do prompt engineering
+last_update:
+   author: Dennis Hume
+sidebar_position: 10
+---
+
+# Prompt Engineering
+
+In this tutorial, you'll build a pipeline with Dagster that:
+
+- Takes an input question
+- Generates prompts to use with Anthropic
+- Validate outputs of AI models and pass outputs across assets
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Basic Python knowledge
+- Python 3.9+ installed on your system. Refer to the [Installation guide](/getting-started/installation) for information.
+</details>
+
+
+## Step 1: Set up your Dagster environment
+
+First, set up a new Dagster project.
+
+1. Within the Dagster repo, navigate to the project:
```

Comment:
It seems basic, but step 1 should probably be "clone the Dagster repo", with a link to the repo